doc/api/process.md

```diff
@@ -1890,6 +1898,20 @@ The `process.version` property returns the Node.js version string.
 console.log(`Version: ${process.version}`);
 ```
 
+## process.computedVersion
```

Comment:
I'd prefer dropping this off `process.release` also to avoid adding anything new to the `process` object.